src/library_pthread.js

```diff
@@ -404,6 +413,10 @@ var LibraryPThread = {
     {{{ makeSetValue('__num_logical_cores', 0, 'cores', 'i32') }}};
   },
 
+  emscripten_transfer_canvas: function() {
+    Module['canvasTransferRequested'] = true;
```

Comment:
Just to consider all possibilities, if we were to make this a non-public thing, how would we do it then? Also does it actually benefit from being public? I could see a use case where it could be set in shell's html file directly for a trivial case, but eh.